### DISCUSSION TITLE
What is the proper way to link the project for the local development?

### DISCUSSION THREAD

**Discussion by mbektimirov** [system-ui/theme-ui#2383] [Q&A] [answered]
I'm trying it with `pnpm link` and getting erros like `Module not found: Can't resolve 'theme-ui/jsx-dev-runtime'` or inability to resolve type declarations. It also links `@theme-ui/monorepo` as a single package - maybe this one is the problem?

Thanks.

**Comment by hasparus** [comment, marked as answer]
Yeah, you don't really want to link the monorepo package. I'd building and then linking after going to package directories, but I'm usually just doing `pnpm pack` and installing from a tarball.
